schedulers/managed-airflow-mwaa/main.tf

```diff
@@ -149,7 +149,18 @@ locals {
     users = [{
       name = "mwaa"
       user = {
-        token = data.aws_eks_cluster_auth.this.token
+        exec = {
+          apiVersion = "client.authentication.k8s.io/v1beta1"
+          command = "aws"
+          args = [
+            "--region",
+            local.region,
+            "eks",
+            "get-token",
+            "--cluster-name",
+            local.name
+          ]
+        }
```

Comment:
Does it fail without using `exec` ? If not we will keep the data source `data.aws_eks_cluster_auth.this.token`

Reply:
@vara-bonthu the token works but will expire after 1h. I go to MWAA eks doc and find they will the above way. I've tested. thanks.
https://docs.aws.amazon.com/mwaa/latest/userguide/mwaa-eks-example.html#eksctl-kube-config